src/Personal/actionCreator.js

```diff
@@ -0,0 +1,64 @@
+import { BACKEND_URL } from 'react-native-dotenv';
+
+export const actionType = {
+  SIGNUP_START: 'SIGNUP_START',
+  SIGNUP_FAILED: 'SIGNUP_FAILED',
+  SIGNUP_SUCCESSFUL: 'SIGNUP_SUCCESSFUL',
+  LOGOUT_SUCCESSFUL: 'LOGOUT_SUCCESSFUL',
```

Comment:
the logout cannot fail, so there is no point to call this LOGOUT_SUCCESSFUL, it van be simply LOGOUT